app/src/main/java/agersant/polaris/PolarisPlaybackService.java

```diff
@@ -90,18 +126,25 @@ public void onCreate() {
 			public void onReceive(Context context, Intent intent) {
 				switch (intent.getAction()) {
 					case PolarisPlayer.PLAYBACK_ERROR:
+						stopDataUpdates();
```

Comment:
Should we also call updateMediaSessionState(NONE) here? Not sure what the most correct behavior is.

Reply:
There is a `STATE_ERROR` I am using now.